### PR TITLE
Bugfix to compute G1 correctly

### PR DESCRIPTION
**Pull request by jzwar**
We were just lucky that it worked so far. I think this should solve the issue

## Checklists
* [ ] Documentations are up-to-date.
* [ ] Added example(s)
* [ ] Added test(s)

### CODE REVIEW

**Review by j042 — APPROVED** [review]
Did it work so far since all the multipatches were well aligned?
So with the normals, you don't have to worry about having a correct partial derivative - is this the fix?

**Comment by jzwar** [comment]
So far we did not check G1, but something stronger. As we required, that adjacent patches hat same normals on the surface

> Did it work so far since all the multipatches were well aligned?
> So with the normals, you don't have to worry about having a correct partial derivative - is this the fix?

It's a weaker condition

**Comment by j042** [comment]
For para_dim==1, this is practically the same, and para_dim==2, it is a different behavior, correct?

**Comment by j042** [comment]
Ok, after discussion, we conclude that this is G1 in a broad sense. Title was just confusing, as it wasn't wrong before

File: src/py/py_multipatch.cpp [hunk 1]
```diff
@@ -833,13 +844,12 @@ int AddBoundariesFromContinuity(const py::list& boundary_splines,
...
-                               const int& patch_id,
-                               const int& face_id) -> std::vector<double> {
+  auto normal_vector = [&cpp_spline_list, &para_dim_, &dim_](
+                           const int& patch_id,
+                           const int& face_id) -> std::vector<double> {
     // init return value (are default initialized to 0)
     std::vector<double> para_coord(para_dim_), bounds(2 * para_dim_),
-        tangential_vector(dim_);
-    std::vector<int> orders(para_dim_);
+        normal_vector(dim_), jacobian(dim_ * para_dim_);
```
Comment (j042) [inline src/py/py_multipatch.cpp:852]: Based on no warning/error, I guess this doesn't necessarily count as "over shadowing" variable?

**Pull request merged**